docs/_sass/_blog.scss

```diff
@@ -32,6 +32,16 @@
     text-decoration: none;
     background-color: darken(#EDF2F6,10);
   }
+
+}
+
+.post-list-thumb {
+  margin-bottom: 3rem;
+  width: 100%;
+}
+.post-image {
```

Comment:
let's aim for consistency in our codebase by adding an extra line between styling.